WalletWasabi/Helpers/Constants.cs

```diff
@@ -96,6 +96,9 @@ public static class Constants
 
 	public static readonly string[] UserAgents = new[]
 	{
+		"/Satoshi:0.23.1/",
+		"/Satoshi:0.22.1/",
+		"/Satoshi:0.24.0.1/",
 		"/Satoshi:0.23.0/",
 		"/Satoshi:0.22.0/",
```

Comment:
This is how these user agents are called AFAIK:

```suggestion
		"/Satoshi:23.1.0/",
		"/Satoshi:24.0.1/",
		"/Satoshi:22.1.0/",
		"/Satoshi:23.0.0/",
		"/Satoshi:22.0.0/",
```

Check https://bitcoincore.org/bin/ and https://bitnodes.io/nodes/

Reply:
right, I continued on current format...